sdk/core/azure-core/azure/core/pipeline/transport/__init__.py

```diff
@@ -158,3 +158,11 @@ def __getattr__(name):
                 pass  # Aiohttp not installed
     except (ImportError, SyntaxError):
         pass  # Asynchronous pipelines not supported.
+
+# see https://pyodide.org/en/stable/usage/faq.html#how-to-detect-that-code-is-run-with-pyodide
+if "pyodide" in sys.modules:
+    from ._pyodide import PyodideTransport, PyodideTransportResponse
```

Comment:
I think we want to load them when we are running in pyodide, otherwise you'd have to import the transport from
`azure.core.pipeline.transport._pyodide`

Reply:
spoke to @kashifkhan. we agreed to remove the import in the `__init__.py` file and rename `_pyodide.py` to `pyodide.py`.

Reply:
spoke to @laurent and decided to revert to `_pyodide.py` and use the new `__init__.py` to lazy import the classes.